### PR TITLE
Upgrade meeting assistant to gsuite add-on

### PR DESCRIPTION
**Pull request by sqrrrl**
Updates manifest + adds homepage experience to work across host apps.

### CODE REVIEW

File: meeting-assistant/src/Main.js [hunk 1]
```diff
@@ -118,6 +132,21 @@ function dispatchActionInternal(event, optErrorHandler) {
...
+ * @return {Card|ActionResponse|UnivseralActionResponse} optional card or action response to render
+ */
+function addOnErrorHandler(err) {
+  var card = buildErrorCard({
+    exception: err,
+    showStackTrace: DEBUG,
+  });
+  return [card];
+}
+
```
Comment (asrivas) [inline meeting-assistant/src/Main.js]: trivial nit

File: meeting-assistant/src/ActionHandlers.js [hunk 2]
```diff
@@ -54,11 +76,20 @@ var ActionHandlers = {
...
-      durationMinutes: parseInt(e.formInput.duration),
-      startHour: parseInt(e.formInput.start),
-      endHour: parseInt(e.formInput.end),
+      emailAddresses: emailAddresses,
+      durationMinutes: parseInt(formInputs.duration[''].stringInputs.value[0]),
+      startHour: parseInt(formInputs.start[''].stringInputs.value[0]),
+      endHour: parseInt(formInputs.end[''].stringInputs.value[0]),
     });
 
     // Validate time ranges -- start must be befor end
```
Comment (asrivas) [inline meeting-assistant/src/ActionHandlers.js]: can you fix this comment while you're here

File: meeting-assistant/src/ActionHandlers.js [hunk 3]
```diff
@@ -54,11 +76,20 @@ var ActionHandlers = {
   findTimes: function(e) {
     var deadlineMonitor = buildDeadlineMonitor(DEFAULT_DEADLINE_SECONDS);
     var settings = getSettingsForUser();
```
Comment (asrivas) [inline meeting-assistant/src/ActionHandlers.js]: is this settings var actually used?

Comment (sqrrrl) [reply meeting-assistant/src/ActionHandlers.js]: It is, but further down. Moved line closer to actual use.

**Pull request merged**